Functions.Templates/Templates/SqlOutputBinding-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": {
+    "text": "SQL output binding",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "3000"
+  },
+  "description": {
+    "text": "A C# function that takes a list of rows and upserts them into the user table",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "3001"
+  },
+  "order": 400,
+  "icon": "vs-2017.3/functionF.png",
+  "learnMoreLink": "https://go.microsoft.com/fwlink/?linkid=848145",
```

Comment:
Maybe these should link to the relevant input/output binding docs instead?

Reply:
aka.ms/sqlbindings